Examples/Examples/FeatureFormExampleView.swift

```diff
@@ -147,70 +168,204 @@ private extension URL {
 /// The model class for the form example view
 @MainActor
 class Model: ObservableObject {
-    /// The feature form.
-    @Published var featureForm: FeatureForm? {
+    /// Feature form workflow states.
+    enum State {
+        /// No feature is being edited.
+        case idle
+        /// A feature form is in use.
+        case editing(FeatureForm)
+        /// The form is being checked for validation errors.
+        case validating(FeatureForm)
+        /// Edits are being committed to the local geodatabase.
+        case finishingEdits(FeatureForm)
+        /// Edits are being applied to the remote service.
+        case applyingEdits(FeatureForm)
+        
```

Comment:
Was there a reason for the empty line?
```suggestion
```

Reply:
Just to form a separation between the linear states and the two others that can pop up at any time.